src/components/Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import { getCommentsForReview } from "../utils/api.js";
+
+const Comments = ({ review_id }) => {
+	const [comments, setComments] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
+
+	useEffect(() => {
+		setIsLoading(true);
+		getCommentsForReview(review_id)
+			.then(({ data }) => {
+				setComments(data.comments);
+				setIsLoading(false);
+			})
+			.catch((error) => {
```

Comment:
If you're going to catch the error, you should probably show the user something of worth regarding this